setup.py

```diff
@@ -31,6 +31,7 @@
         'redis>=3.0.0',
         'funcy>=1.8,<2.0',
         'six>=1.4.0',
+        'dill>=0.3.3',
```

Comment:
No new dependencies. Most people don't need it.

Reply:
How you see picklng not supported by `pickle` lib curried methods without any new dependency? Maybe I don't see right way for it,

Reply:
I see above - doing it in few days. Thank's!